src/rmind/components/episode.py

```diff
@@ -35,6 +35,7 @@ class Modality(StrEnum):
     CONTINUOUS = auto()
     DISCRETE = auto()
     SPECIAL = auto()
+    POINTS = auto()
```

Comment:
1. not sure if worth extending `Modality` for this (vs keeping under continuous/discrete)
2. if so, is `points` a good name for this?

@sandhawalia wdyt

Reply:
i'm okay adding a new modality but maybe rename the type as `CONTEXT` which could be natural language instructios or waypoints.